app/main.py

```diff
@@ -1,3 +1,14 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    dict_of_args = {}
+
+    def inner(*args: tuple) -> Callable:
+        if args in dict_of_args.keys():
```

Comment:
```suggestion
        if args in dict_of_args:
```

It will result in same as 

```if args in dict_of_args.keys()```